<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>166</width>
     <height>24</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditYear">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMonth">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDay">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelHeader">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>用阿拉伯数字输入合法的年月日。</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelYear">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMonth">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDay">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
